run_linux.go

```diff
@@ -76,7 +76,7 @@ func (b *Builder) Run(command []string, options RunOptions) error {
 	logrus.Debugf("using %q to hold bundle data", path)
 	defer func() {
 		if err2 := os.RemoveAll(path); err2 != nil {
-			logrus.Error(err2)
+			b.Logger.Error(err2)
```

Comment:
```suggestion
			options.Logger.Error(err2)
```
Also needs to create a logger for `options.Logger` if it isn't set.